README.md

```diff
@@ -1,1110 +1,18 @@
-
-```
-curl demo.nats.io:8222/varz
-```
-
-To enable the monitoring server, start the NATS server with the monitoring flag `-m` (or `-ms`) and specify the monitoring port.
-
-Monitoring options
-
-    -m, --http_port PORT             HTTP PORT for monitoring
-    -ms,--https_port PORT            Use HTTPS PORT for monitoring (requires TLS cert and key)
-
-To enable monitoring via the configuration file, use `host:port` (there is no explicit configuration flag for the monitoring interface).
+[![License][License-Image]][License-Url] [![FOSSA Status][Fossa-Image]][Fossa-Url] [![ReportCard][ReportCard-Image]][ReportCard-Url] [![Build][Build-Status-Image]][Build-Status-Url] [![Release][Release-Image]][Release-Url] [![Coverage][Coverage-Image]][Coverage-Url] [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1895/badge)](https://bestpractices.coreinfrastructure.org/projects/1895)
 
-For example, running the `nats-server -m 8222` command, you should see that the NATS server starts with the HTTP monitoring port enabled. To view the monitoring home page, go to <a href="http://localhost:8222/" target="_blank">http://localhost:8222/</a>.
+## Documentation
 
-```
-[70450] 2018/08/29 12:48:30.819682 [INF] Starting nats-server version 1.3.0
-[70450] 2018/08/29 12:48:30.819750 [INF] Git commit [not set]
-[70450] 2018/08/29 12:48:30.819918 [INF] Starting http monitor on 0.0.0.0:8222
-[70450] 2018/08/29 12:48:30.819960 [INF] Listening for client connections on 0.0.0.0:4222
-[70450] 2018/08/29 12:48:30.819964 [INF] Server is ready
-```
+* [Official documentation](https://nats.io/documentation)
+* [FAQ](https://nats.io/documentation/faq/)
+* Watch [a video overview of NATS](https://www.youtube.com/watch?v=sm63oAVPqAM) to learn more about its origin story and design philosophy
 
 ## Community and Contributing
 
-NATS has a vibrant and friendly community.  If you are interested in connecting with other NATS users or contributing, read about our [community](https://nats.io/community/) on [NATS.io](https://nats.io/).
-
-### NATS Office Hours
-
-NATS Office Hours will be on hiatus for the US summer season. Please join our [Slack channel](https://join.slack.com/t/natsio/shared_invite/enQtMzE2NDkxNDI2NTE1LTc5ZDEzYTkwYWZkYWQ5YjY1MzBjMWZmYzA5OGQxMzlkMGQzMjYxNGM3MWYxMjNiYmNjNzIwMTVjMWE2ZDgxZGM) or [Google Group](https://groups.google.com/forum/#!forum/natsio) to chat with our maintainers.
-
+NATS has a vibrant and friendly community.  If you are interested in connecting with other NATS users or contributing, read about our [community](https://nats.io/community/) on [NATS.io](https://nats.io/). To chat with our maintainers you can also join our [Slack channel](https://join.slack.com/t/natsio/shared_invite/enQtMzE2NDkxNDI2NTE1LTc5ZDEzYTkwYWZkYWQ5YjY1MzBjMWZmYzA5OGQxMzlkMGQzMjYxNGM3MWYxMjNiYmNjNzIwMTVjMWE2ZDgxZGM) or [Google Group](https://groups.google.com/forum/#!forum/natsio).
```

Comment:
With the contact section, I would remove these two links (slack and google). Would make it all about contributing. So something like:
```
If you are interested in contributing, read about our...
* [Contributing guide](https://nats.io/community/#contribute)
* [Report issues or propose Pull Requests](https://github.com/nats-io)
```

Reply:
updated the section here too